edx_rest_framework_extensions/auth/jwt/decoder.py

```diff
@@ -15,7 +15,7 @@
 
 
 class JwtTokenVersion(object):
-    latest_supported = '1.1.0'
+    latest_supported = '1.2.0'
```

Comment:
1. I thought this is where we were going to comment a mini changelog of past versions, or is that somewhere in edx-platform?
2. Is this what we discussed becoming `default_latest_supported`?

Reply:
I didn't know where we intended to put the comment. I can put it here

I wasn't sure we decided to change `default_latest_supported` based on the slack messages, so I didn't. I can though

Reply:
Actually i'm not going to add a mini changelog. i believe we determined git log was the best place for it